tools/explorer/lib/ontology_wrapper.py

```diff
@@ -42,7 +49,27 @@ def GetFieldsForTypeName(self,
     Returns:
             result_fields: a list of StandardField tuples
     """
-    pass
+    entity_type = self.universe.entity_type_universe.GetEntityType(
+        namespace,
+        entity_type_name
+    )
+    entity_type_fields = [
+        ETF(
+            namespace_name=field.field.namespace,
+            standard_field_name=field.field.field[0:],
```

Comment:
wow, this seems like this violates LoD and is very confusing to read. can we provide a getter function for standard_field_name on a field?

Reply:
@kswang2400 that's exactly what I'm doing here. I'm converting the optwrapper and field parts named tuples into actual objects which do have getter functions like GetStandardFieldName() and such.

see:
https://github.com/google/digitalbuildings/blob/6fa3ee2ecaa752e186173209272b7a93c917fc80/tools/validators/ontology_validator/yamlformat/validator/entity_type_lib.py#L34

Reply:
got it, yeah this is better! :)

is there a reason to do `field[0:]`? isn't it the same as just field?

Reply:
@trav3711 oh actually can you change this, the linter does not accept it. The message I got, this is more than 3 lines, not great for readability.
```
 ETF(
            namespace_name=field.field.namespace,
            standard_field_name=field.field.field[0:],
            ```